sa/storage-authority.go

```diff
@@ -213,6 +213,28 @@ func (ssa *SQLStorageAuthority) GetAuthorization(id string) (authz core.Authoriz
 	return
 }
 
+// GetAuthorizationsByDomain obtains all authorizations for a domain name
+func (ssa *SQLStorageAuthority) GetAuthorizationsByDomain(domain core.AcmeIdentifier) ([]core.Authorization, error) {
+	tx, err := ssa.dbMap.Begin()
+	if err != nil {
+		return nil, err
+	}
+	auths := []core.Authorization{}
+	_, err = tx.Select(&auths, "SELECT * FROM authorizations WHERE identifier = :identifier", map[string]interface{}{"indentifier": domain})
```

Comment:
*identifier. I think this query should fail based on that typo, which means: maybe we need more tests?